src/config/content-configurations.tsx

```diff
@@ -0,0 +1,23 @@
+import TeacherEditionA from "../icons/teacher_edition_A.svg";
+import ExclamationA from "../icons/exclamation_A.svg";
+
+export interface IConfiguration {
+  icon: SvgrComponent;
+  label: string;
+  styleClassName: string;
+}
+
+export const ContentConfigurations: {[index: string]: IConfiguration} = {
+  teacherTip: {
+    icon: ExclamationA,
```

Comment:
I mentioned that somewhere above - if it was `Icon` instead, it'd be a bit easier to use it in JSX as a react component.

Reply:
Done.